### PR TITLE
Add more samples in standardized format

### PR DESCRIPTION
**Pull request by hsubox76**
Adding the following samples in the new standardized sample format.

See https://docs.google.com/spreadsheets/d/10X--DI0YPL3QfEMObe387rYg0t4-kuv33yKn2TIal7E/edit?resourcekey=0-kfy1Ry4YAZh6O4DltLjV9w&gid=0#gid=0 (internal)

filename|region tag
-- | --
function_calling | function_calling
controlled_generation | json_controlled_generation
controlled_generation | json_no_schema
model_configuration | configure_model
safety_settings | safety_settings
safety_settings | safety_settings_multi
system_instruction | system_instruction

### CODE REVIEW

File: samples/node/safety_settings.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+  const result = await model.generateContent(unsafePrompt);
+
+  try {
+    result.response.text();
+  } catch (e) {
+    console.error(e);
+    console.log(result.response.candidates[0].safetyRatings);
+  }
+  // [END safety_settings]
```
Comment (dlarocque) [inline samples/node/safety_settings.js:49]: This throws an error around half the time for me.

File: samples/node/safety_settings.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    result.response.text();
+  } catch (e) {
+    console.error(e);
+    console.log(result.response.candidates[0].safetyRatings);
+  }
+  // [END safety_settings_multi]
+}
+
+async function run() {
+  // await safetySettings();
```
Comment (dlarocque) [inline samples/node/safety_settings.js:85]: I think we want to un-comment this call?

**Review by dlarocque — CHANGES_REQUESTED** [review]

File: samples/node/safety_settings.js (hunk 2 above)
Comment (hsubox76) [reply samples/node/safety_settings.js]: Oops, fixed.

File: samples/node/safety_settings.js (hunk 1 above)
Comment (hsubox76) [reply samples/node/safety_settings.js]: I made a modification that seems to trigger it every time I've tried, let me know what you think.

**Review by dlarocque — APPROVED** [review]

**Pull request merged**